libnavigation-core/src/main/java/com/mapbox/navigation/core/routealternatives/RouteAlternativesObserver.kt

```diff
@@ -18,10 +16,11 @@ interface RouteAlternativesObserver {
      *
      * @param routeProgress the current route's progress.
      * @param alternatives list of alternative routes, can be empty.
+     *
+     * @return list of indexes that should no longer be considered alternatives.
```

Comment:
Should we expose this as NN is doing? Wondering if forcing clients to return something is a good approach 🤔 

https://github.com/mapbox/mapbox-navigation-android/blob/80f54d6a01820f8f302045b2632da208b836e4c5/qa-test-app/src/main/java/com/mapbox/navigation/qa_test_app/view/AlternativeRouteActivity.kt#L307

Reply:
Yeah I created an issue with NN too. Because we definitely do not want to expose the list in this way.

Reply:
Removed